tests/fields/test_types.py

```diff
@@ -118,6 +141,17 @@ class Type:
         ("json", JSON),
     ]
 
+    if hasattr(models, "GeneratedField"):
```

Comment:
Maybe use `if django.VERSION >= (5, 0)` to make sure any typos in here would not make it pass by mistake?